services/director-v2/tests/conftest.py

```diff
@@ -291,3 +292,42 @@ def mocked_service_awaits_manual_interventions(mocker: MockerFixture) -> None:
         autospec=True,
         return_value=False,
     )
+
+
+@pytest.fixture
+def mock_redis(mocker: MockerFixture) -> None:
+    def _mock_setup(app: FastAPI) -> None:
+        def _mock_client(*args, **kwargs) -> AsyncMock:
+            return AsyncMock()
+
+        mock = AsyncMock()
+        mock.client = _mock_client
+
+        async def on_startup() -> None:
+            app.state.redis_clients_manager = mock
+
+        app.add_event_handler("startup", on_startup)
+
+    mocker.patch(
+        "simcore_service_director_v2.modules.redis.setup", side_effect=_mock_setup
+    )
+
+
+@pytest.fixture
+def mock_exclusive(mock_redis: None, mocker: MockerFixture) -> None:
+    def _mock_exclusive(
```

Comment:
THOUGHT: it might be handy to have this mock in pytest_simcore. exclusive locks is something we are using all over the place and this mock is not trivial

Reply:
Since it depends on the mock_redis fixture, it's a bit more complicated to port. If somone requires it, we shall find a way to port it. For now I will leave it here.